CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,93 @@
+# PrestaShop Code of Conduct
+
+English version 
+
+## Purpose 
+
+This Code of Conduct outlines how to behave to participate in the PrestaShop open source project (the “Project”) as well as prohibited behaviours and reporting procedure for damaging behaviours and/or situations. 
```

Comment:
Behavior is [generally uncountable](http://www.macmillandictionaryblog.com/language-tip-of-the-week-behaviou).

```suggestion
This Code of Conduct outlines how to behave to participate in the PrestaShop open source project (the “Project”) as well as prohibited behaviour and reporting procedure for damaging behaviour and/or situations. 
```